src/Sentry/Laravel/Integration.php

```diff
@@ -192,13 +189,7 @@ public static function sentryTracingMeta(): string
      */
     public static function sentryBaggageMeta(): string
     {
-        $span = SentrySdk::getCurrentHub()->getSpan();
-
-        if ($span === null) {
-            return '';
-        }
-
-        return sprintf('<meta name="baggage" content="%s"/>', $span->toBaggage());
+        return sprintf('<meta name="baggage" content="%s"/>', getBaggage());
```

Comment:
As we'll very likely always get some value here, I think it's OK to omit checks here.